src/morphometrics/segment/segment.py

```diff
@@ -0,0 +1,106 @@
+import glob
+import os
+
+import h5py
+import numpy as np
+import tifffile
+import wget
+import yaml
+from plantseg.pipeline.raw2seg import raw2seg
+from stardist.models import StarDist3D
+
+
+def load_to_disk(preprocessed_image):
```

Comment:
```suggestion
def _save_to_disk(preprocessed_image):
```
I would make this a private function since it is not meant to be used except for within the pipeline. In python we denote this with `_` in front of the variable name.